templates/base.html

```diff
@@ -1,6 +1,5 @@
 <!DOCTYPE html>
 
-<!-- generated. do not manually edit -->
```

Comment:
I think this is intended for the files generated from this template (e.g. https://get.jenkins.io/redhat-stable/ ), and as such needs to be part of the template file (misleading me on first reading).

A better wording could be

```
<!-- Do not manually edit this file, it is generated from https://github.com/jenkinsci/packaging/blob/master/templates/base.html -->